bin/node/cli/Cargo.toml

```diff
@@ -0,0 +1,135 @@
+
+# paint dependencies
+indices = { package = "paint-indices", path = "../../../paint/indices" }
+timestamp = { package = "paint-timestamp", path = "../../../paint/timestamp", default-features = false }
+finality_tracker = { package = "paint-finality-tracker", path = "../../../paint/finality-tracker", default-features = false }
+contracts = { package = "paint-contracts", path = "../../../paint/contracts" }
+system = { package = "paint-system", path = "../../../paint/system" }
+balances = { package = "paint-balances", path = "../../../paint/balances" }
+transaction-payment = { package = "paint-transaction-payment", path = "../../../paint/transaction-payment" }
+support = { package = "paint-support", path = "../../../paint/support", default-features = false }
+im_online = { package = "paint-im-online", path = "../../../paint/im-online", default-features = false }
+
+# node-specific dependencies
+node-runtime = { path = "../runtime" }
+node-rpc = { path = "../rpc" }
+node-primitives = { path = "../primitives" }
+node-executor = { path = "../executor" }
+
+# CLI-specific dependencies
+tokio = { version = "0.1.22", optional = true }
+exit-future = { version = "0.1.4", optional = true }
+substrate-cli = { path = "../../../client/cli", optional = true }
+transaction-factory = { path = "../../../test/utils/transaction-factory", optional = true }
+ctrlc = { version = "3.1.3", features = ["termination"], optional = true }
+
+# WASM-specific dependencies
+libp2p = { version = "0.13.0", default-features = false, optional = true }
+clear_on_drop = { version = "0.2.3", features = ["no_cc"], optional = true }	# Imported just for the `no_cc` feature
+console_error_panic_hook = { version = "0.1.1", optional = true }
+console_log = { version = "0.1.2", optional = true }
+js-sys = { version = "0.3.22", optional = true }
+wasm-bindgen = { version = "0.2.45", optional = true }
+wasm-bindgen-futures = { version = "0.3.22", optional = true }
+kvdb-memorydb = { git = "https://github.com/paritytech/parity-common", rev="b0317f649ab2c665b7987b8475878fc4d2e1f81d", optional = true }
+rand6 = { package = "rand", version = "0.6", features = ["wasm-bindgen"], optional = true }	# Imported just for the `wasm-bindgen` feature
+
+[dev-dependencies]
+keystore = { package = "substrate-keystore", path = "../../../client/keystore" }
+babe = { package = "substrate-consensus-babe", path = "../../../client/consensus/babe", features = ["test-helpers"] }
+consensus-common = { package = "substrate-consensus-common", path = "../../../primitives/consensus/common" }
+service-test = { package = "substrate-service-test", path = "../../../client/service/test" }
+futures03 = { package = "futures-preview", version = "0.3.0-alpha.19" }
+tempfile = "3.1.0"
+
+[build-dependencies]
+substrate-cli = { package = "substrate-cli", path = "../../../client/cli" }
+build-script-utils = { package = "substrate-build-script-utils", path = "../../../client/utils/build-script-utils" }
```

Comment:
This project is missing in the global `Cargo.toml` + should be moved with `wasm-builder`.